branding_company/model/branding_company.py

```diff
@@ -30,6 +29,29 @@ class BrandingCompany(models.Model):
         supporting models and reports.
         """
 
+    def get_partner_branding_company(self, partner_id):
+        """Returns recordset with branding company for partner or False."""
+        if not partner_id:
+            return False
+        partner_model = self.env['res.partner']
+        partner_obj = partner_model.browse(partner_id)
+        return partner_obj and partner_obj.branding_company_id or False
```

Comment:
@NL66278 but then you get an empty browse record of type self._name, which is `branding.company`, but you need an empty browse record of type `res.partner` - otherwise, somebody accessing res.partner fields still get errors because the field must exist in the model for the empty browse record to return something empty